vnext/ReactUWP/Views/TouchEventHandler.cpp

```diff
@@ -158,10 +158,9 @@ void TouchEventHandler::OnPointerConcluded(
   // Only if the view has a Tag can we process this
   int64_t tag;
   winrt::FrameworkElement sourceElement(nullptr);
-  if (!TagFromOriginalSource(args, &tag, &sourceElement))
-    return;
+  if (TagFromOriginalSource(args, &tag, &sourceElement))
+    UpdateReactPointer(m_pointers[pointerIndex], args, sourceElement);
 
-  UpdateReactPointer(m_pointers[pointerIndex], args, sourceElement);
   DispatchTouchEvent(eventType, pointerIndex);
```

Comment:
>DispatchTouchEvent [](start = 2, length = 18)

Do you still want to dispatch event if tag not found?

Reply:
I was going to let it still go so that js code expecting any down would have a corresponding up get sent, start/end of some interaction.